### DISCUSSION TITLE
learning reference

### DISCUSSION THREAD

**Discussion by sepide-aghaei** [permitio/fastapi_websocket_pubsub#23] [Q&A] [answered]
Hi everyone. Is there any course or reference for learning this repository?Thank you

**Comment by orweis** [comment, marked as answer]
Hi @sepide-aghaei,
There's this video review/tutorial from PyCon you can use:
https://www.youtube.com/watch?v=KP7tPeKhT3o

And of course the Readme file for the repository.
Also these folders are a good place to start for  [examples](https://github.com/authorizon/fastapi_websocket_pubsub/blob/master/examples) and [tests](https://github.com/authorizon/fastapi_websocket_pubsub/blob/master/tests) 

And please don't hesitate to ask questions

**Reply by sepide-aghaei** [reply]
Thank you very much
